common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1493,7 +1493,11 @@ public enum PUBLIC_SHARE_VISIBILITY { samePrimaryDomain, all, none };
     
     // ZCS-11349: Toggle off/on fallback to ldap search
     public static final KnownKey zimbra_gal_fallback_ldap_search_enabled = KnownKey.newKey(true);
-
+    
+    // ZCS-11602: Notification support for quarantined devices
+    public static final KnownKey zimbra_mobile_mdm_notification_email_body = KnownKey.newKey("Following is the list of quarantined devices ");
```

Comment:
This is not what LC is for. Move:
https://github.com/Zimbra/zm-mailbox/tree/develop/store-conf/conf/msgs
https://github.com/Zimbra/zm-mailbox/blob/develop/store-conf/conf/msgs/ZsMsg.properties
https://github.com/Zimbra/zm-mailbox/blob/develop/common/src/java/com/zimbra/common/util/L10nUtil.java

Reply:
Reason of this put in LC is it is needed to be configurable.